decidim-core/app/mailers/decidim/reported_mailer.rb

```diff
@@ -48,6 +50,10 @@ def report_url
       @report_url ||= EngineRouter.admin_proxy(@participatory_space).moderation_report_url(host: @organization.host, moderation_id: @report.moderation.id, id: @report.id)
     end
 
+    def proposal_component_url
```

Comment:
```suggestion
    def resource_admin_url
```
This method should be more generic. Not all reported content is a proposal.
Moreover, not all reported content has an admin link.
I'd propose here to return this url only if the resource is administrable. Then, in the mailer view show the "admin this resource" link if it exists.

```ruby
def resource_admin_url
  @resource_admin_url ||= Decidim::ResourceLocatorPresenter.new(@reportable).admin_url
end
```

And modify ResourceLocatorPresenter addin the method `admin_url`